### PR TITLE
[onewire] fix DS2438 Vad measurement

### LINKED ISSUE #4761: [Onewire] Filter bad VAD values for iButtonLink Multisensor devices
## Expected Behavior
iButtonLink's MS-TL and MS-TV devices have a tendency to report VAD values that are over 10 when they should be reporting as 0 instead. These values should be translated to 0 so that accurate light, humidity, etc are reported correctly.

## Current Behavior
The Onewire binding is using these incorrect values, which is causing their associated values to be wildly inaccurate. My MS-TL devices, for example, will report a light level of well over 500,000 lux in a pitch black room.

## Possible Solution
When pulling the VAD values from either an MS-TL or MS-TV device, values that are clearly out of spec should be changed to 0 before using them to calculate other values. [This discussion on the owfs mailing list](https://sourceforge.net/p/owfs/mailman/message/22188323/) suggests changing anything over 10 for both devices (which is what I did with the onewire 1.x binding), however I think limiting it to 5 instead of 10 for the MS-TL would make more sense since the iButtonLink's [manual for the multisensor devices](https://cdn.shopify.com/s/files/1/0164/3524/files/MultiSensor_Manual_v1.5.pdf?15845834050373852166) says 5V is the max value it will report.

## Steps to Reproduce (for Bugs)

1. Add an MS-TL device to a onewire network is place it in a sealed box to prevent any light from entering
2. After a while, the device will start reporting extremely large lux values

## Context

This has forced me to disable to or rewrite several rules that use the light or voltage levels I'm monitoring, such as turning off devices or dimming devices when it's dark in a room.

## Your Environment

* Version used: openHAB 2.5.0 Build 1511, Onewire 2.5.0.201901280820
* Environment name and version: Oracle JDK 1.8.0.181
* Operating System and version: Gentoo Linux

### PR DESCRIPTION
**Pull request by J-N-K**
Fixes #4761 

DS2438 has a range of 0-10V for the analog input. For some unknown reason values above 10 are reported where they should be 0 instead.

Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.kvv/.project [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.kvv/.project]: Does this file belong here?

File: addons/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/device/DS2438.java [hunk 2]
```diff
@@ -270,13 +276,15 @@ public void refresh(OwBaseBridgeHandler bridgeHandler, Boolean forcedRefresh) th
...
-                                    Math.pow(10, (65 / 7.5) - (47 / 7.5) * (Vcc / ((DecimalType) light).doubleValue())),
-                                    SmartHomeUnits.LUX);
-                            callback.postUpdate(CHANNEL_LIGHT, light);
+                        double measured = ((DecimalType) bridgeHandler.readDecimalType(sensorId, voltageParameter))
+                                .doubleValue();
+                        if (measured > 10.0) {
+                            // workaround bug in DS2438
+                            measured = 0.0;
                         }
+                        light = new QuantityType<Illuminance>(Math.pow(10, (65 / 7.5) - (47 / 7.5) * (Vcc / measured)),
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/device/DS2438.java]: The division by `measured` will result in an `ArithmeticException` "division by zero`.

File: addons/binding/org.openhab.binding.kvv/.project (hunk 1 above)
Comment (J-N-K) [reply addons/binding/org.openhab.binding.kvv/.project]: Surely not.

File: addons/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/device/DS2438.java (hunk 2 above)
Comment (J-N-K) [reply addons/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/device/DS2438.java]: The formula is correct. I‘ll add a range-check for the measured value (>0 && <=10).

**Pull request merged**